website/source/docs/providers/aws/r/api_gateway_deployment.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: "aws"
+page_title: "AWS: aws_api_gateway_deployment"
+sidebar_current: "docs-aws-resource-api-gateway-deployment"
+description: |-
+  Provides an API Gateway Deployment.
+---
+
+# aws\_api\_gateway\_deployment
+
+Provides an API Gateway Deployment.
+
+**Note:** Depends on having `aws_api_gateway_method` inside your rest api. To ensure this
```

Comment:
Also I think we should add the `depends_on` into the example below, so that we people copy-paste the example, they realise it. It can be either just imaginary method which doesn't exist in the example or you can add the definition for `aws_api_gateway_method` too - up 2 you.